init.fish

```diff
@@ -46,6 +46,8 @@ __git_abbr gcm        git commit -m
 __git_abbr gcam       git commit -a -m
 __git_abbr gscam      git commit -S -a -m
 __git_abbr gcl        git clone
+__git_abbr gclean     git clean -di
+__git_abbr gclean!    git clean -xdf
```

Comment:
```suggestion
__git_abbr gclean!    git clean -dfx
```

to match order of git help, please use same order for the new gclean!! abbreviation

Reply:
So that's it. I got it now, it's alphabetical. Didn't notice that before 😅 . 
Fixed.